charts/linkerd2/README.md

```diff
@@ -1,263 +1,124 @@
+| global.proxyInit.closeWaitTimeoutSecs | int | `0` |  |
+| global.proxyInit.ignoreInboundPorts | string | `""` |  |
+| global.proxyInit.ignoreOutboundPorts | string | `""` |  |
+| global.proxyInit.image.name | string | `"ghcr.io/linkerd/proxy-init"` |  |
+| global.proxyInit.image.pullPolicy | string | `"IfNotPresent"` |  |
+| global.proxyInit.image.version | string | `"v1.3.6"` |  |
+| global.proxyInit.resources.cpu.limit | string | `"100m"` |  |
+| global.proxyInit.resources.cpu.request | string | `"10m"` |  |
+| global.proxyInit.resources.memory.limit | string | `"50Mi"` |  |
+| global.proxyInit.resources.memory.request | string | `"10Mi"` |  |
+| global.proxyInit.xtMountPath.mountPath | string | `"/run"` |  |
+| global.proxyInit.xtMountPath.name | string | `"linkerd-proxy-init-xtables-lock"` |  |
+| global.proxyInjectAnnotation | string | `"linkerd.io/inject"` |  |
+| global.proxyInjectDisabled | string | `"disabled"` |  |
+| global.workloadNamespaceLabel | string | `"linkerd.io/workload-ns"` |  |
+| grafana.enabled | bool | `true` |  |
+| heartbeatSchedule | string | `"0 0 * * *"` |  |
+| identity.issuer.clockSkewAllowance | string | `"20s"` |  |
+| identity.issuer.crtExpiry | string | `nil` |  |
+| identity.issuer.crtExpiryAnnotation | string | `"linkerd.io/identity-issuer-expiry"` |  |
+| identity.issuer.issuanceLifetime | string | `"86400s"` |  |
+| identity.issuer.scheme | string | `"linkerd.io/tls"` |  |
+| identity.issuer.tls.crtPEM | string | `""` |  |
+| identity.issuer.tls.keyPEM | string | `""` |  |
+| installNamespace | bool | `true` |  |
+| nodeSelector."beta.kubernetes.io/os" | string | `"linux"` |  |
+| omitWebhookSideEffects | bool | `false` |  |
+| profileValidator.caBundle | string | `""` |  |
+| profileValidator.crtPEM | string | `""` |  |
+| profileValidator.externalSecret | bool | `false` |  |
+| profileValidator.keyPEM | string | `""` |  |
+| prometheus.enabled | bool | `true` |  |
+| proxyInjector.caBundle | string | `""` |  |
+| proxyInjector.crtPEM | string | `""` |  |
+| proxyInjector.externalSecret | bool | `false` |  |
+| proxyInjector.keyPEM | string | `""` |  |
+| tap.caBundle | string | `""` |  |
+| tap.crtPEM | string | `""` |  |
+| tap.externalSecret | bool | `false` |  |
+| tap.keyPEM | string | `""` |  |
+| tracing.enabled | bool | `false` |  |
+| webImage | string | `"ghcr.io/linkerd/web"` |  |
+| webhookFailurePolicy | string | `"Ignore"` |  |
 
-Linkerd is a *service mesh*, designed to give platform-wide observability,
-reliability, and security without requiring configuration or code changes.
-
-Linkerd is a Cloud Native Computing Foundation ([CNCF][cncf]) project.
-
-## Quickstart and documentation
-
-You can run Linkerd on any Kubernetes 1.13+ cluster in a matter of seconds. See
-the [Linkerd Getting Started Guide][getting-started] for how.
-
-For more comprehensive documentation, start with the [Linkerd
-docs][linkerd-docs].
-
-## Prerequisite: identity certificates
-
```

Comment:
What can we do to avoid losing all these docs? This is surfaced in places like https://artifacthub.io/packages/helm/linkerd2/linkerd2 so it's important to keep these instructions

Reply:
Added templating files for solving this issue. Should do the trick!